packages/quantic/force-app/main/default/lwc/quanticGeneratedAnswer/templates/generatedAnswer.html

```diff
@@ -1,41 +1,56 @@
 <template>
   <template lwc:if={shouldDisplayGeneratedAnswer}>
-    <div
-      data-cy="generated-answer__card"
-      class="slds-box"
-    >
-      <div
-        class="slds-var-m-bottom_medium slds-grid generated-answer__card-header"
-      >
+    <div data-cy="generated-answer__card" class="slds-box">
+      <div class="generated-answer__card-header slds-grid">
         <span
           class="generated-answer__badge slds-var-p-around_x-small slds-text-title_bold"
           >{labels.generatedAnswerForYou}</span
         >
-        <c-quantic-feedback
-          state={feedbackState}
-          onlike={handleLike}
-          ondislike={handleDislike}
-          like-icon-name="utility:like"
-          like-label={labels.thisAnswerWasHelpful}
-          dislike-icon-name="utility:dislike"
-          dislike-label={labels.thisAnswerWasNotHelpful}
-          size="x-small"
-          question=""
-          hide-explain-why-button
-          hide-labels
-          data-cy="generated-answer__feedback"
-        ></c-quantic-feedback>
-      </div>
-      <div data-cy="generated-answer__content" class={generatedContentClass}>
-        {answer}
+        <div class="slds-grid">
+          <template lwc:if={shouldDisplayFeedback}>
+            <c-quantic-feedback
+              state={feedbackState}
+              onlike={handleLike}
+              ondislike={handleDislike}
+              like-icon-name="utility:like"
+              like-label={labels.thisAnswerWasHelpful}
+              dislike-icon-name="utility:dislike"
+              dislike-label={labels.thisAnswerWasNotHelpful}
+              size="x-small"
+              question=""
+              hide-explain-why-button
+              hide-labels
+              data-cy="generated-answer__feedback"
+            ></c-quantic-feedback>
+          </template>
+          <div class="slds-var-m-left_medium">
+            <lightning-input
+              data-cy="generated-answer__toggle-button"
+              type="toggle"
+              onchange={toggleGeneratedAnswer}
+              checked={isVisible}
+              variant="label-hidden"
+              message-toggle-active=""
+              message-toggle-inactive=""
+              label={toggleGeneratedAnswerLabel}
```

Comment:
According to [these guidelines](https://www.w3.org/WAI/ARIA/apg/patterns/switch/#:~:text=Important%3A%20it%20is%20critical%20the%20label%20on%20a%20switch%20does%20not%20change%20when%20its%20state%20changes.), the switch label should always be the same.

Reply:
Today I learned! thank you